01_host/06_anv/approval-voting.adoc

```diff
@@ -183,14 +183,13 @@ t_3 larr sf "append"(t_2, "'core'",c_i)\
 t_4 larr "append"(t_3, "'vrf-nm-pk'", p_k)\
 t_5 larr "meta-ad"(t_4, "'VRFHash'", "False")\
 t_6 larr "meta-ad"(t_5, 64_("le"), "True")\
-i larr larr "prf"(t_6, "False")\
+i "prf"(t_6, "False")\
 o = s_k * i\
-(p,x) larr sf "dleq_prove"(t_6, i)
+p larr sf "dleq_prove"(t_6, i)
 ++++
 
-The resulting values stem:[h] and stem:[p] are the VRF input/output pair
-respectively the VRF proof (<<defn-vrf-proof>>). stem:["dleq_prove"] is
-described in <<defn-vrf-dleq-prove>>.
+The resulting value stem:[p] is the VRF proof (<<defn-vrf-proof>>).
+stem:["dleq_prove"] is described in <<defn-vrf-dleq-prove>>.
```

Comment:
Normally a VRF proof consists of two component (output + proof), so we should describe both here.

Reply:
`o` is the output. I updated this section which explains how those values are used:

![image](https://user-images.githubusercontent.com/42901763/161257106-2240cc36-1090-431d-8461-f18818b4f7e2.png)